frontend/src/routes/(authenticated)/(dashboards)/admin/+page.svelte

```diff
@@ -102,24 +163,56 @@
                 <span class="i-mdi-sort-ascending text-xl align-[-5px] ml-2" />
               </th>
               <th>{$t('admin_dashboard.column_type')}</th>
+              <th />
             </tr>
           </thead>
           <tbody>
-            {#each projects as project}
+            {#each projects as project (project.id)}
               <tr>
                 <td>
-                  <a class="link" href={`/project/${project.code}`}>
-                    {project.name}
-                  </a>
+                  {#if project.deletedDate}
+                    <span class="flex gap-2 text-error">
+                      {project.name}
+                      <TrashIcon pale />
+                    </span>
+                  {:else}
+                    <a class="link" href={`/project/${project.code}`}>
+                      {project.name}
+                    </a>
+                  {/if}
                 </td>
                 <td>{project.code}</td>
                 <td>TODO</td>
                 <td>
-                  <FormatDate date={project.lastCommit} />
+                  {#if project.deletedDate}
+                    <span class="text-error">
+                      <FormatDate date={project.deletedDate} />
```

Comment:
might be good to prefix this with something to indicate that this isn't the last time data was pushed to the repo but it's actually the deleted date.

Reply:
IMO making the text red communicates that sufficiently.
After all, the column is called "Last Change" not "Last Sync" or "Last Push".